Gemfile

```diff
@@ -27,3 +27,4 @@ gem 'omniauth-github', :github => 'intridea/omniauth-github'
 gem 'gems', '~> 0.8.3'
 gem 'kaminari', '~> 0.16.3'
 gem 'sdoc', '~> 0.4.0', group: :doc
+gem 'mandrill-api'
```

Comment:
@linatodi I think we can remove this gem since we are not going to use Mandrill's api from our app, at least so far. We will use mandrill to send emails from heroku and the only thing we need to achieve that is to configure the smtp_sttings of ActionMailer and this gem is not necessary to do that.